### DISCUSSION TITLE
How can I create a VM Generation V2 image?

### DISCUSSION THREAD

**Discussion by devlife** [actions/virtual-environments#11152] [General questions]
I'm running the following but it seems to always create an image that is VM Generation `1`. However, the VM Scale Set that I'll be using will be using V2 VMs. How can I create a `V2` image?

`GenerateResourcesAndImage -SubscriptionId XXXX -ResourceGroupName YYYYY -ImageGenerationRepositoryRoot "$pwd" -ImageType Ubuntu2204 -AzureLocation "East US"`

![image](https://github.com/user-attachments/assets/c631b988-5ed9-49f0-bfaf-5f18603f4ed6)

**Comment by rybal06** [comment]
https://github.com/actions/runner-images/discussions/6780

**Reply by devlife** [reply]
I was hoping there was something else. Thanks.

**Comment by subir0071** [comment]
`V2` generation is in the product roadmap.
However, we are unable to put a release date for this feature.

**Reply by PetrPospisil** [reply]
https://github.com/actions/runner-images/pull/13543
